invokeai/app/api/dependencies.py

```diff
@@ -122,6 +124,14 @@ def initialize(config: InvokeAIAppConfig, event_handler_id: int, logger: Logger
 
         db.clean()
 
+        # clean up latents - this is a temporary measure to respect users' disk space
+        deleted_latents_count = 0
+        for latents in Path(latents_path).glob("*"):
+            if latents.is_file():
+                latents.unlink()
```

Comment:
Does unlink actually delete also?

Reply:
Yes

Reply:
Please, consider moving these files outside the "output" folders. Also, cleanliness is next to godliness, don't just remove the files, delete parent directory instead. No one wants empty folders either.

follow-up on Discord channel:
https://discord.com/channels/1020123559063990373/1156344991292063825/1156344991292063825

Reply:
- They *are* outputs, so they will be in the outputs folder, which is primarily an application-facing folder.
- The `latents/` directory is used by the application, so it needs to exist.
- We are using modern python's `pathlib` as a general rule. `pathlib` does not delete non-empty directories. Globbing the files and deleting each one is a fine way to handle this.